src/stores/options.json

```diff
@@ -92,11 +92,37 @@
         "itemType": "section",
         "label": "Fenix"
       },
+      {
+        "label": "Any Fenix or Firefox Preview Activity",
```

Comment:
this criterion seems to return an empty result set – is the key right, or is the criterion not staged yet?

Reply:
That was a mistake in the staged data. I've fixed the problem now and it's populated.